src/main/java/org/sagebionetworks/bridge/services/AppConfigService.java

```diff
@@ -206,6 +256,7 @@ public AppConfig createAppConfig(String appId, AppConfig appConfig) {
         newAppConfig.setSchemaReferences(appConfig.getSchemaReferences());
         newAppConfig.setConfigReferences(appConfig.getConfigReferences());
         newAppConfig.setFileReferences(appConfig.getFileReferences());
+        newAppConfig.setAssessmentReferences(appConfig.getAssessmentReferences());
```

Comment:
Unit test to make sure that assessement references are being marshalled when creating?

Reply:
Added